package.json

```diff
@@ -7,13 +7,15 @@
     "access": "public"
   },
   "description": "A CLI tool for managing cursor workflows",
-  "main": "bin/cursor-companion.mjs",
+  "main": "dist/src/index.js",
```

Comment:
Please take note of this change, it's important for the npm registry. Test this still works.

Reply:
@jeremytoce